src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -29,15 +28,21 @@ export class SearchMenu extends React.Component {
     this.state = {
       userInput: '',
       suggestions: [],
+      highlightedIndex: null,
     };
   }
+
   componentDidUpdate(prevProps, prevState) {
     const { userInput } = this.state;
+    const { searchTypeaheadEnabled } = this.props;
 
+    // if userInput has changed, fetch suggestions for the typeahead experience
     const inputChanged = prevState.userInput !== userInput;
-    if (inputChanged) {
+    if (inputChanged && searchTypeaheadEnabled) {
```

Comment:
Oh nice! This will cut down the requests quite a bit when typeahead isn't enabled